server/dummyControllers/requestController.js

```diff
@@ -36,6 +37,44 @@ class RequestController {
       });
     }
   }
+
+  /**
+ * it GET all requests
+ * @param {string} req
+ * @param {string} res
+ * @returns {object} object
+ */
+  static getARequest(req, res) {
+    const id = parseInt(req.params.id, 10);
```

Comment:
check for edge cases do you what happens when you parseInt a string?

Reply:
ok